File: Curso Programação Orientada a Objetos/PHP com POO/Aula12/Peixe.php
<?php

require_once 'Animal.php';

class Peixe extends Animal {
    //Atributos
    private $corEscama;
    
    //Métodos
    public function soltarBolha() {
        echo "<p>Peixe está soltando bolha!</p>";
    }
    
    //Métodos Sobrepostos
    public function locomover() {
        echo "<p>Nadando</p>";
    }

    public function alimentar() {
        echo "<p>Comendo alga</p>";
    }

    public function emitirSom() {
        echo "<p>Som de Peixe</p>";
    }
    
    //Getters
    public function getCorEscama() {
        return $this->corEscama;
    }

    //Setters
    public function setCorEscama($corEscama) {
        $this->corEscama = $corEscama;
    }


}

File: Curso Programação Orientada a Objetos/Java com POO/Aula12e13/src/aula12e13/Peixe.java
package aula12e13;

public class Peixe extends Animal {
    //Atributos
    private String corEscama;
    
    //Métodos
    public void soltarBolha() {
        System.out.println("Peixe está soltando bolhas!");
    }
    
    //Métodos Sobrepostos
    @Override
    public void locomover() {
        System.out.println("Nadando");
    }

    @Override
    public void alimentar() {
        System.out.println("Comendo algas");
    }

    @Override
    public void emitirSom() {
        System.out.println("Som de Peixe");
    }
    
    //Getters e Setters
    //corEscama
    public String getCorEscama() {
        return corEscama;
    }
    public void setCorEscama(String corEscama) {
        this.corEscama = corEscama;
    }
    
    
}
